assets/src/apps/delivery/layouts/deck/components/FeedbackContainer.tsx

```diff
@@ -42,7 +44,7 @@ const FeedbackContainer: React.FC<FeedbackContainerProps> = ({
   return (
     <div
       className={`feedbackContainer rowRestriction ${isLegacyTheme ? 'columnRestriction' : ''}`}
-      style={{ top: '525px' }}
+      style={{ top: '525px', ...style }}
```

Comment:
looks like it's optional, maybe above needs `style = {}` just in case so it doesn't error on the spread here

Reply:
👍